Section 2 - Read, Generate, Modify Congiruations/attributes.tf

```diff
@@ -14,6 +14,46 @@ output "eip" {
 
 resource "aws_s3_bucket" "mys3" {
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure bucket policy is enforced with least privileges for all AWS S3 buckets**
 
**Vulnerability**
Overly permissive Amazon S3 buckets may lead to unauthorized access.
 
**Remediation**
Ensure access to Amazon S3 buckets are tightly controlled with least priviledge. Implementing least privilege access is fundamental in reducing security risk and the impact that could result from errors or malicious intent.

source: tenablecs-